integrationtests/src/test/java/io/kroxylicious/proxy/TlsIT.java

```diff
@@ -53,7 +53,7 @@ class TlsIT extends BaseIT {
     private static final HostPort PROXY_ADDRESS = HostPort.parse("localhost:9192");
     private static final ClusterNetworkAddressConfigProviderDefinition CONFIG_PROVIDER_DEFINITION = new ClusterNetworkAddressConfigProviderDefinitionBuilder(
             "PortPerBroker").withConfig("bootstrapAddress", PROXY_ADDRESS)
-            .build();
+                    .build();
```

Comment:
this block looks a bit weird:
```
private static final ClusterNetworkAddressConfigProviderDefinition CONFIG_PROVIDER_DEFINITION = new ClusterNetworkAddressConfigProviderDefinitionBuilder(
            "PortPerBroker").withConfig("bootstrapAddress", PROXY_ADDRESS)
                    .build();
```

Reply:
hmm, yeah. Tweaking it also leads to a fight between intellij and Maven.... Will have to open Eclipse again to see if I can get it right there.

Reply:
Not sure why this is being changed, but we have a line length limit of 170 (this hasn't changed) which is reached somewhere in the string `PortPerBroker`. There are no good answers  for this one.

The best I've managed is 
```
 private static final ClusterNetworkAddressConfigProviderDefinition CONFIG_PROVIDER_DEFINITION = new ClusterNetworkAddressConfigProviderDefinitionBuilder(
            "PortPerBroker")
            .withConfig("bootstrapAddress", PROXY_ADDRESS)
            .build();
```